Building from adam-core module directory fails to generate Scala code for sql package

```bash
adam-core master
$ mvn clean compile
...
[INFO]
[INFO] --- build-helper-maven-plugin:1.12:add-source (add-source) @ adam-core-spark2_2.11 ---
[INFO] Source directory: adam/adam-core/src/main/scala added.
[INFO] Source directory: adam/adam-core/target/generated-sources/src/main/scala added.
[INFO]
[INFO] --- exec-maven-plugin:1.5.0:java (generate-scala-products) @ adam-core-spark2_2.11 ---
[INFO]
[INFO] --- exec-maven-plugin:1.5.0:java (generate-scala-projection-fields) @ adam-core-spark2_2.11 ---
[INFO]
...
[INFO] --- scala-maven-plugin:3.2.2:compile (scala-compile-first) @ adam-core-spark2_2.11 ---
...
[ERROR] adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/VariantRDD.scala:42: error: object Variant is not a member of package org.bdgenomics.adam.sql
[ERROR] import org.bdgenomics.adam.sql.{ Variant => VariantProduct }
[ERROR]        ^
...

adam-core master
$ cd ..

adam master
$ mvn clean compile
...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
```

--- a/adam-core/pom.xml
+++ b/adam-core/pom.xml
@@ -114,7 +114,7 @@
                 <argument>org.bdgenomics.formats.avro.Variant</argument>
                 <argument>org.bdgenomics.formats.avro.VariantAnnotation</argument>
                 <argument>org.bdgenomics.formats.avro.VariantCallingAnnotations</argument>
-                <argument>adam-core/target/generated-sources/src/main/scala/org/bdgenomics/adam/sql/Schemas.scala</argument>
+                <argument>${project.basedir}/target/generated-sources/src/main/scala/org/bdgenomics/adam/sql/Schemas.scala</argument>
               </arguments>
               <classpathScope>compile</classpathScope>
             </configuration>
@@ -146,7 +146,7 @@
                 <argument>org.bdgenomics.formats.avro.Variant</argument>
                 <argument>org.bdgenomics.formats.avro.VariantAnnotation</argument>
                 <argument>org.bdgenomics.formats.avro.VariantCallingAnnotations</argument>
-                <argument>adam-core/target/generated-sources/src/main/scala/org/bdgenomics/adam/projections/Enums.scala</argument>
+                <argument>${project.basedir}/target/generated-sources/src/main/scala/org/bdgenomics/adam/projections/Enums.scala</argument>
               </arguments>
               <classpathScope>compile</classpathScope>
             </configuration>

